File: src/main/java/net/softbell/bsh/domain/repository/MemberRepo.java
package net.softbell.bsh.domain.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import net.softbell.bsh.domain.entity.Member;

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : 회원 리포지토리 인터페이스
 */
@Repository
public interface MemberRepo extends JpaRepository<Member, Long>
{
	Member findByUserId(String userId);
}

File: src/main/kotlin/net/softbell/bsh/domain/repository/MemberRepo.kt
package net.softbell.bsh.domain.repository

import net.softbell.bsh.domain.entity.Member
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

/**
 * @author : Bell(bell@softbell.net)
 * @description : 회원 리포지토리 인터페이스
 */
@Repository
interface MemberRepo : JpaRepository<Member, Long> {
    fun findByUserId(userId: String): Member?
}